docs/natspec-format.rst

```diff
@@ -49,12 +49,13 @@ The following example shows a contract and a function using all available tags.
 .. code:: Solidity
 
     // SPDX-License-Identifier: GPL-3.0
-    pragma solidity >=0.6.12 <0.9.0;
+    pragma solidity >=0.8.2;
```

Comment:
Typically, we always add `<` next breaking release, since the contract may break.
```suggestion
    pragma solidity >=0.8.2 < 0.9.0;
```